chsdi/lib/helpers.py

```diff
@@ -193,7 +197,7 @@ def getOperator(values):
             return operator
         return ''
 
-    regEx = r'(\w+\s(?:ilike|not ilike)\s(?:\'%)[^\'%]+(?:%\'))|(\w+\s(?:=|\!=|>=|<=|>|<)\s[^\s]+)'
+    regEx = r'(\w+\s(?:ilike|not ilike)\s(?:\'%)[^\'%]+(?:%\'))|(\w+\s(?:=|\!=|>=|<=|>|<)\s[^\s]+)|(\w+\s(?:is|is not)\s(?:null))'
```

Comment:
Just tested the resulting where is an empty string, meaning that we don't use any filter when it should return a bad request. so I think this could be improved. I'll adapt.